cmds/cpu/cpu.go

```diff
@@ -170,12 +175,32 @@ func newCPU(host string, args ...string) error {
 
 func main() {
 	flags()
+	host := ""
 	args := flag.Args()
-	if len(args) == 0 {
-		usage()
+	a := ""
+
+	if *mdnsEnabled {
```

Comment:
This probably needs to be in the client package.

Reply:
@rminnich - pushed movement of mdns into client module -- this relies on you being able to set network to mdns and using hostname to specify selection/prioritization criteria. This pass muster or do you want me to try and parse URI to specify mdns -- and then what kind of syntax:
mdns://arch=arm64,mem>16G:cpucount>16
or
mdns://_ncpu._tcp(arch=arm64,mem>16G:cpucount>16)
or
mdns!_ncpu._tcp!_arch=arm64,mem>16G:cpucount>16
(or something else)

Of course, this may be the next level of abstraction - maybe just mdns to start with as hostname?